plugins/node/opentelemetry-instrumentation-bunyan/src/instrumentation.ts

```diff
@@ -96,7 +96,7 @@ export class BunyanInstrumentation extends InstrumentationBase<BunyanInstrumenta
   }
 
   override setConfig(config: BunyanInstrumentationConfig = {}) {
-    this._config = { ...DEFAULT_CONFIG, ...config };
```

Comment:
This did override the `enabled` config and actually disabled the instrumentation, this went unnoticed until the change from https://github.com/open-telemetry/opentelemetry-js/commit/0ee398d1e3f84059dc648597aa1b2f7be368f262 which now uses this method instead of setting it directly.